client/i18n/locales/chinese/translations.json

```
@@ -1,444 +1,444 @@
-    "ask-for-help": "Ask for Help",
-    "create-post": "Create a help post on the forum",
-    "cancel": "Cancel",
-    "reset-lesson": "Reset this lesson",
-    "run": "Run",
-    "run-test": "Run the Tests",
-    "reset": "Reset",
-    "reset-code": "Reset All Code",
-    "help": "Help",
-    "get-help": "Get Help",
-    "watch-video": "Watch a Video",
-    "resubscribe": "You can click here to resubscribe",
-    "click-here": "Click here to sign in",
-    "save": "Save",
-    "no-thanks": "No thanks",
-    "yes-please": "Yes please",
-    "update-email": "Update my Email",
-    "verify-email": "Verify Email",
-    "submit-and-go": "Submit and go to next challenge",
-    "go-to-next": "Go to next challenge",
-    "ask-later": "Ask me later"
+    "logged-in-cta-btn": "开始学习（免费）",
+    "logged-out-cta-btn": "登录以保存你的学习进度（免费）",
+    "view-curriculum": "查看课程",
+    "first-lesson": "访问第一个课程",
+    "close": "关闭",
+    "edit": "编辑",
+    "show-code": "显示代码",
+    "show-solution": "显示解决方案",
+    "frontend": "前端",
+    "backend": "后端",
+    "view": "查看",
+    "show-cert": "显示认证",
+    "claim-cert": "申请认证",
+    "save-progress": "保存进度",
+    "accepted-honesty": "你已经接受我们的《学术诚信条例》",
+    "agree": "同意",
+    "save-portfolio": "保存这个作品集项目",
+    "remove-portfolio": "移除这个作品集项目",
+    "add-portfolio": "增加一个新的作品集项目",
+    "download-data": "下载你的数据",
+    "public": "公开",
+    "private": "仅自己可见",
+    "off": "关闭",
+    "on": "打开",
+    "sign-in": "登录",
+    "sign-out": "退出",
+    "curriculum": "课程",
+    "forum": "论坛",
+    "profile": "个人资料",
+    "update-settings": "更新我的账号设置",
+    "sign-me-out": "退出登录 freeCodeCamp",
+    "flag-user": "标记该用户的账号为滥用",
+    "current-challenge": "访问目前的挑战",
+    "try-again": "重试",
+    "menu": "菜单",
+    "settings": "设置",
+    "take-me": "访问课程",
+    "check-answer": "查看你的答案",
+    "get-hint": "获得提示",
+    "ask-for-help": "请求帮助",
+    "create-post": "在论坛发帖请求帮助",
+    "cancel": "取消",
+    "reset-lesson": "重置课程",
+    "run": "运行",
+    "run-test": "运行测试",
+    "reset": "重置",
+    "reset-code": "重置所有代码",
+    "help": "帮助",
+    "get-help": "获得帮助",
+    "watch-video": "观看视频",
+    "resubscribe": "你可以点击这里重新订阅",
+    "click-here": "点击这里登陆",
+    "save": "保存",
+    "no-thanks": "不用，谢谢",
+    "yes-please": "好的",
+    "update-email": "更新我的邮箱",
+    "verify-email": "确认邮箱",
+    "submit-and-go": "提交并访问下一个挑战",
+    "go-to-next": "访问下一个挑战",
+    "ask-later": "稍后问我"
   },
   "landing": {
-    "big-heading-1": "Learn to code at home.",
-    "big-heading-2": "Build projects.",
-    "big-heading-3": "Earn certifications.",
-    "h2-heading": "Since 2014, more than 40,000 freeCodeCamp.org graduates have gotten jobs at tech companies including:",
-    "hero-img-description" : "freeCodeCamp students at a local study group in South Korea.",
-    "as-seen-in": "As seen in:",
+    "big-heading-1": "在家学习编程",
+    "big-heading-2": "构建项目",
+    "big-heading-3": "获得认证",
+    "h2-heading": "自 2014 年以来，已有超过 40,000 名 freeCodeCamp 学员入职科技公司，例如：",
```

```suggestion
    "h2-heading": "自 2014 年以来，已有超过 40,000 名 freeCodeCamp.org 学员入职科技公司，例如：",
```

Not sure, english has it.